pages/linux/protontricks.md

```diff
@@ -18,3 +18,11 @@
 - Run a command within a game's installation directory:
 
 `protontricks -c {{command}} {{appid}}`
+
+- List all installed games:
+
+`protontricks -l`
+
+- Search for a game's App ID by entering it's name:
```

Comment:
**fixup:** "it's" is used as a contraction of "it is", "its" is the possessive form. We can also avoid this entirely:

```suggestion
- Search for a game's App ID by name:
```